### PR TITLE
LPS-171793 | GIVI Commerce Task | Test Automation | Master

### PR DESCRIPTION
**Pull request by evanilson-santana**
https://issues.liferay.com/browse/LPS-171793

### CODE REVIEW

**Comment by evanilson-santana** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 1]
```diff
@@ -185,6 +185,104 @@ definition {
...
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "test@liferay.com",
+				fromAddress = "test@liferay.com",
+				fromName = "Test test",
+				name = "Notification Template",
+				subject = "Subject");
+
+			Button.clickSave();
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:207]: Let's change this to PortletEntry.save(); per Brian Lee's comment

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 2]
```diff
@@ -185,6 +185,104 @@ definition {
...
+			ObjectAdmin.addObjectActionViaUI(
+				actionName = "Notification action when order status update",
+				active = "true",
+				inputExpression = "orderStatus == 0",
+				notificationName = "Notification Template",
+				thenAction = "Notification",
+				whenAction = "On Order Status Update");
+		}
+
+		task ("And when: an account, minium site and an order is created") {
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:226]: I think it makes more sense to have these as And given. All of these things you are creating is setup.

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 3]
```diff
@@ -185,6 +185,104 @@ definition {
...
+
+		task ("Then: The action is triggered and the notification is sent") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Queue");
+
+			AssertElementPresent(
+				key_objectName = "CommerceOrder",
+				locator1 = "ObjectNotifications#OBJECT_QUEUE_NAME");
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:282]: Have you tried testing this with a mock email server running? We should have coverage to see if the email is being sent but that can probably be covered in ObjectNotfications.testcase. This assertion should be fine

**Review by epark18 — COMMENTED** [review]
Good job on this, very minor changes needed. Also remember to add the test results on the ticket.

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase (hunk 3 above)
Comment (evanilson-santana) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:282]: > Have you tried testing this with a mock email server running? We should have coverage to see if the email is being sent but that can probably be covered in ObjectNotfications.testcase. This assertion should be fine

Not yet, for this case, we just need to test if the notification was sent. Thanks!

**Comment by epark18** [comment]
ci:forward

**Pull request closed without merging**